docs/team/Qiaoran.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: Meng Qiaoran's Project Portfolio Page
+---
+
+### Project: Duke The Market
+
+Duke The Market is a desktop contact management system application
+used for managing customer contacts in department stores. The user interacts with it using a CLI,
+and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+
+Given below are my contributions to the project.
+
+* **Code contributed**: to be added soon
+
+* **Enhancements implemented**:
+    * to be added soon
+
+* **Documentation**:
+    * Contributions to the User Guide:
+        * to be added soon
+    * Contributions to the Developer Guide:
+        * to be added soon
+
+* **Contributions to team-based tasks**:
+    * to be added soon
+
+* **Community**:
+    * Review/Mentoring contributions:
+        * to be added soon
+    * Contributions beyond the project team:
+        * to be added soon
```

Comment:
As specified under the [failing CI checks](https://github.com/AY2223S1-CS2103-F09-2/tp/pull/19/checks), I think we will need to add an extra newline character to the end of every file (Hence, why this file failed the CI check as well).

Reply:
thank you! I will add the newline character to this file.